src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs

```diff
@@ -34,22 +34,22 @@ public KestrelEngine(ILibraryManager libraryManager, ServiceContext context)
                 {
                     var architecture = IntPtr.Size == 4
                         ? "x86"
-                        : "amd64";
+                        : "x64";
```

Comment:
When your native support comes in we should be able to get rid of this completely.